.rtlcssrc

```diff
@@ -1,10 +1,6 @@
 {
-    "options": {
-        "preserveComments": true,
-        "preserveDirectives": false,
-        "swapLeftRightInUrl": true,
-        "swapLtrRtlInUrl": true,
-        "swapWestEastInUrl": false,
-        "autoRename": false
-    }
+	"options": {
+		"processUrls": { "atrule": true, "decl": false }
```

Comment:
'atrule' affects URLs in at-rules such as
`@import url("example-ltr.css");`

'decl' affects urls in declarations:
`background: url( 'something-rtl.png');`